OrbitGl/GraphTrack.h

```diff
@@ -14,17 +14,20 @@ class TimeGraph;
 
 class GraphTrack : public Track {
  public:
-  explicit GraphTrack(TimeGraph* time_graph);
+  explicit GraphTrack(TimeGraph* time_graph, uint64_t graph_id);
   [[nodiscard]] Type GetType() const override { return kGraphTrack; }
   void Draw(GlCanvas* canvas, PickingMode /*picking_mode*/) override;
   [[nodiscard]] float GetHeight() const override;
+  void AddValue(double value, uint64_t time);
+  double GetValueAtTime(uint64_t time, double default_value = 0) const;
```

Comment:
[[nodiscard]]

Reply:
Let's discuss the use of [[nodiscard]] with the team.

Reply:
Done.